website/pages/en/developing/supported-networks.mdx

```diff
@@ -4,35 +4,37 @@ title: Supported Networks
 
 import { getSupportedNetworks } from '@/src/getSupportedNetworks'
 
-## The Graph's Decentralized Network
-
-The following networks are supported on The Graph's Decentralized Network:
-
-<ListUnordered>
-  {getSupportedNetworks('studio').map((network) => (
-    <li key={network.cliName}>
-      <CodeInline>{network.cliName}</CodeInline>
-      {network.cliName === 'mainnet' ? ` (${network.name})` : null}
-      {network.beta ? '*' : null}
-    </li>
+Here is a table showing the networks that The Graph supports, whether it is on the hosted service, Subgraph Studio, or the decentralized network:
```

Comment:
Not sure about this text. Aren't the rows actually "chains" rather than "networks"? Should we rename this entire page?

Reply:
Agree about the text - I think the table is self-explanatory, so we can probably drop it?
On chains vs. networks, that is a general naming issue we have, but I am ok with keeping network here as `network` is what it is called in the subgraph manifest

Reply:
Thank you! I removed the paragraph in https://github.com/graphprotocol/docs/pull/405/commits/94c8f0f04996747a70e04361f5169ee5d7ec953c.